README.md

```diff
@@ -8,8 +8,19 @@ Use the bash script provided and provide a version number.
 source build_and_push_image.sh ${MAJOR}.${MINOR}.${PATCH}
 ```
 
+The above code snippet will not work until you have your AWS credentials. Please request access from Michael G.
```

Comment:
```suggestion
The above code snippet will not work until you have your AWS credentials.
```
Making this change allows these docs to remain useful even if the procedure for obtaining AWS creds changes in the future. For example, the process could be delegated to other staff members or we might create an automated process which provisions credentials ahead of time.

Reply:
Makes sense.